examples/cloud-composer-examples/composer_http_post_example/ephemeral_dataproc_spark_dag.py

```diff
@@ -64,7 +64,7 @@
     # POST to the REST API.
     # Alternatively, we could set this to yesterday and the dag will be triggered upon upload to the
     # dag folder.
-    'start_date': datetime.utcnow(),
+    'start_date': datetime(2020, 1, 1),
```

Comment:
this fixes DAG immediately being marked success without scheduling tasks in newer airflow versions.